services/bower/bower-version.tester.js

```diff
@@ -44,10 +49,15 @@ t.create('Version label should be `no releases` if no stable version')
   .expectBadge({ label: 'bower', message: 'no releases' })
 
 t.create('Version label should be `no releases` if no pre-release')
-  .get('/vpre/bootstrap.json')
+  .get('/v/bootstrap.json?include_prereleases')
   .intercept(nock =>
     nock('https://libraries.io')
       .get('/api/bower/bootstrap')
       .reply(200, { normalized_licenses: [], latest_release_number: null })
   )
   .expectBadge({ label: 'bower', message: 'no releases' })
+
+t.create('Version (legacy redirect: vpre)')
+  .get('/vpre/bootstrap.json', { followRedirect: false })
```

Comment:
We need `{ followRedirect: false }` if we want to test redirects, because `followRedirect` is `true` by default. But do we actually need following redirects in our tests? In my opinion we can change `followRedirect` to `false` by default in all service tests. What's your opinion?

Reply:
I'm going to leave this one for now as it needs a full test suite run. In the abstract, I think you're probably right but I'd want to do a full service test run with it off to see what happens.